README.md

```diff
@@ -2,94 +2,93 @@
+App Service is Azure's fully-managed Platform as a Service (PaaS) that let's you
+deploy and scale web, mobile, and API apps. Use the Azure App Service extension
+for VS Code to quickly create, manage, and deploy your websites.
 
-Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app-service-extension/getting-started) to get started with deploying a Node.js Express app to Azure App Service on Linux.
+Visit the [wiki](https://github.com/Microsoft/vscode-azureappservice/wiki) for
+information on advanced features such as Remote Debugging.
 
-## Features
+## Installation
 
-* Create new web apps/deployment slots
-* Deploy to your web apps/deployment slots
-* Stream your logs to VS Code's output window
-* Add and edit environment variables (app settings)
-* Browse sites across all of your Azure subscriptions
-* Browse to the Azure Portal for advanced tasks, such as scaling
+1. Download and install the Azure App Service extension for Visual Studio Code
+    > If you're interested in deploying single page web apps or progressive web apps (something without an express server), install the [Azure Storage extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestorage)
+2. Wait for the extension to finish installing then reload Visual Studio Code when prompted
+3. Once complete, you'll see an Azure icon in the Activity Bar
+    > If your activity bar is hidden, you won't be able to access the extension. Show the Activity Bar by clicking View > Appearance > Show Activity Bar
+4. Sign in to your Azure Account by clicking Sign in to Azure…
+    >  If you don't already have an Azure Account, click "Create a Free Azure Account" or you can [try Azure for free](https://code.visualstudio.com/tryappservice/?utm_source=appservice-extension) for an hour
 
-![Deploy to Web App](resources/WebApp_Deploy.png)
+## Deploy your first Node.js app to Azure
 
-* Start, stop, and restart the web app/deployment slot
-* Swap deployment slots
-* View and edit web app settings
-* View web app log stream
-* View and edit a web app's remote files
-  * To view a file, click on it in the explorer.
-  * To edit, make edits in the editor and save it.  When prompted to upload the file, click 'Upload'.
-  * CAUTION: Manually editing your Web App's files could cause unexpected behavior.
-* View a web app's log files
-* View web app's current and previous git deployments
-* Connect web apps to Cosmos databases via Connections
+Once you are signed into your Azure account and you have your app open in Visual
+Studio Code, click the deploy button in the Azure App Service explorer - it's
+the blue up arrow - to deploy your app.
 
-![Web App Log Stream](resources/WebApp_LogStream.png)
+<img src="https://user-images.githubusercontent.com/1186948/50742590-49250000-11c2-11e9-85ff-c5f4c9352e3c.png" width="300">
```

Comment:
So we don't have to version the images

Reply:
How/where are you uploading the images? In case other people want to do the same thing, but also wondering how to ensure they're not deleted on accident

Reply:
That...seems hacky. I'm not sure I trust those images to stay up forever. I also like having the images in source control with the PR. Can you expand on "So we don't have to version the images"?

Reply:
done